test/integration/eu-west-1/k8s-qa19/services/terragrunt.hcl

```diff
@@ -56,9 +56,10 @@ inputs = {
   # Platform Flux CD
   # --------------------------------------------------
 
-  // platform_fluxcd_deploy       = true
-  // platform_fluxcd_repo_name    = "platform-manifests-qa"
-  // platform_fluxcd_github_owner = "dfds"
+  platform_fluxcd_deploy       = true
+  platform_fluxcd_release_tag  = "v0.8.2"
```

Comment:
Don't we generally favour 'latest', i.e. not define, for QA?

Reply:
Yes.  Can I actually put latest there though or do I just omit the value entirely?